eng/Versions.props

```diff
@@ -105,6 +105,7 @@
     <MicrosoftDotNetCommonProjectTemplates31PackageVersion>$(MicrosoftDotNetCommonItemTemplates31PackageVersion)</MicrosoftDotNetCommonProjectTemplates31PackageVersion>
     <MicrosoftDotNetTestProjectTemplates31PackageVersion>$(MicrosoftDotNetTestProjectTemplates50PackageVersion)</MicrosoftDotNetTestProjectTemplates31PackageVersion>
     <AspNetCorePackageVersionFor31Templates>3.1.7</AspNetCorePackageVersionFor31Templates>
```

Comment:
Please update this line to `3.1.8` and [line 122](https://github.com/dotnet/installer/pull/8426/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR122) to `2.1.22` i.e. correct all the project templates from dotnet/aspnetcore. This will help me w/ dotnet/aspnetcore-internal#3676

Reply:
Done.